src/sentry/runner/commands/presenters/slackpresenter.py

```diff
@@ -66,7 +66,12 @@ def is_slack_enabled():
             raise
 
     def flush(self) -> None:
+        region: Optional[str] = settings.SENTRY_REGION
+        if not region:
+            region = settings.CUSTOMER_ID
```

Comment:
What is this is None as well ? Would the webhook break if you pass None ?

Reply:
Webhook will not break, it will default to sending the message without region defined. I can reconfigure this behavior in eng-pipes as needed, but I think thats fine.